README.md

```diff
@@ -6,6 +6,7 @@ Receipt scanner extracts information from your PDF or image receipts.
 [![NPM version](https://img.shields.io/npm/v/receipt-scanner.svg)](https://www.npmjs.com/package/receipt-scanner)
```

Comment:
Remove this, or don't add code climate badge. We should not have this many badges.